content/docs/command-reference/import.md

```diff
@@ -70,19 +70,19 @@ enable DVC efficiently determining whether the local copy is out of date.
 To actually [version the data](/doc/tutorials/get-started/data-versioning),
 `git add` (and `git commit`) the import `.dvc` file.
 
-Note that `dvc repro` doesn't check or update import `.dvc` files (see
-`dvc freeze`), use `dvc update` to bring the import up to date from the data
-source.
+Note that `dvc repro` doesn't check or update import `.dvc` files by default
+(see `dvc freeze`), use `dvc update` to bring the import up to date from the
+data source.
 
 Also note that chained imports (importing data that was imported into the source
 repo at `url`) are not supported.
 
 ## Options
 
-- `-o <path>`, `--out <path>` - specify a path to the desired location in the
-  workspace to place the downloaded file or directory (instead of using the
-  current working directory). Directories specified in the path must already
-  exist, otherwise this command will fail.
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to place
+  the downloaded file or directory (instead of using the current working
```

Comment:
is instead of the current directory? or is it instead of a default file name in the current directory? can we phrase it a bit simple, explicit. `(by default it creates the file in the directory you run the command in and it has a name ...)`

Reply:
It says "to place" (the data in) but sure, I can rephrase in a clearer way... ⌛

Reply:
Done.